libs/wire-api/src/Wire/API/User/Client.hs

```diff
@@ -373,17 +379,17 @@ instance Swagger.ToSchema UserClientsFull where
 
 instance ToJSON UserClientsFull where
   toJSON =
-    toJSON . Map.foldrWithKey' fn Map.empty . userClientsFull
+    toJSON . Map.foldrWithKey' f Map.empty . userClientsFull
```

Comment:
it's not a big deal, ignore this if you like.  but in general (a) i don't like single-character identifiers, even locally; (b) i want us all to avoid edit wars.

Reply:
You don't like single-character identifiers, but two-character ones are ok? That's a bit of a strange stance. But sure, I'll revert it.